flow/scripts/global_route.tcl

```diff
@@ -11,6 +11,9 @@ if {[info exist env(FASTROUTE_TCL)]} {
 } else {
   set_global_routing_layer_adjustment $env(MIN_ROUTING_LAYER)-$env(MAX_ROUTING_LAYER) 0.5
   set_routing_layers -signal $env(MIN_ROUTING_LAYER)-$env(MAX_ROUTING_LAYER)
+  if {[info exist env(MACRO_EXTENSION)]} {
```

Comment:
Nit: Break this out into a separate commit before this commit.

Reply:
Rewriting the git history is tricky so I'd like to avoid this.  I'll keep it in mind for the future.

Reply:
No, it is easy, like juggling, 6 hours a day for 6 months, anyone can do it :-)

Split a commit:

git rebase -i origin/master
git reset -p HEAD^
git commit --amend
# answer yes to the hunks you want to break out
git add .
git commit -m "broken out commit"
git rebase --continue

Reply:
I went through the adventure and I think it worked :)  Thanks